files/saslauthd-postfix

```diff
@@ -0,0 +1,44 @@
+# {{ ansible_managed }}
+
+# Should saslauthd run automatically on startup? (default: no)
+START=yes
+
+# Description of this saslauthd instance. Recommended.
+# (suggestion: SASL Authentication Daemon)
+DESC="SASL Authentication Daemon for Postfix"
+
+# Short name of this saslauthd instance. Strongly recommended.
+# (suggestion: saslauthd)
+NAME="saslauthd-pstfx"
```

Comment:
Why abbreviate "postfix"?

Reply:
Because the name can be 15 characters at most.